### ISSUE TITLE
API for Chrome Web Store

### ISSUE THREAD

**Issue by ishu3101** [mdamien/chrome-extensions-archive#9]
Do you have plans to create an api for Chrome Web Store like the itunes store?

**Comment by mdamien** [comment]
yes, I'm gonna make the sql database be able to be queried directly via #2 and #4

**Comment by ishu3101** [comment]
So is it going to be similar to the itunes store api and will the api return data as json?

**Comment by mdamien** [comment]
json for sure but I don't know yet the query language (sql, graphql, REST,..). Why do you want an API ?

**Comment by ishu3101** [comment]
Yes, I have been looking for an api for the Chrome Web Store but could not find it. When is the api going to be live?

**Comment by mdamien** [comment]
I haven't started working on it so there's no ETA, maybe you can try to do it ? Getting the data should be simple with the code currently in this repo
